.storybook/manager-head.html

```diff
@@ -191,7 +191,7 @@
     svgElem.setAttributeNS(null, "viewBox", `0 0 ${boxWidth * 2} ${boxHeight * 2}`);
     svgElem.setAttributeNS(null, "width", boxWidth);
     svgElem.setAttributeNS(null, "height", boxHeight);
-    svgElem.setAttributeNS(null, "role", "image");
+    svgElem.setAttributeNS(null, "role", "img");
```

Comment:
```suggestion
    svgElem.setAttributeNS(null, "role", "img");
    svgElem.setAttributeNS(null, "aria-label", " to ");
```

Fix for AI tool aria-label errors.